### PR TITLE
SDK-704: we only need to submit identity_docs for creating KYC

### PR DESCRIPTION
**Pull request by hailefrollo**
- Is the destination branch correct & intended? ✅ 
- Is the correct JIRA ticket referenced? ✅ 
- Is the correct API docs/Confluence Page referenced? ✅ 
- Have all the AC's in the ticket passed? ✅ 
- Have you followed proper naming conventions in the code? ✅ 
- Has the code been properly covered with unit tests? ✅ 
- Have you regenerated the mock DBs, if the datamodel was modified ✅ 
- Have you added migration tests (if applicable)? ✅ 
- Have the unit tests passed locally (if applicable)? ✅ 
- Have you addressed the code warnings if any? ✅ 
- Have you tested the API and made sure the documentation is up-to-date? ✅ 
- Have you added documentation for all public methods & types ✅

### CODE REVIEW

File: Sources/FrolloSDK/KYC/KYC.swift [hunk 1]
```diff
@@ -62,8 +62,10 @@ public class KYC: ResponseHandler {
      */
     
     public func submitKYC(userKYC: UserKYC, completion: @escaping (Result<UserKYC, Error>) -> Void) {
+        // SDK-704: now we only need to submit identity documents for KYC
+        let newKYC = UserKYC(identityDocuments: userKYC.identityDocuments)
```
Comment (dipeshfrollo) [inline Sources/FrolloSDK/KYC/KYC.swift:66]: why not just send UserKYC with only identityDocuments from client app and directly pass that to API?

File: Sources/FrolloSDK/Model/Transient/UserKYC.swift [hunk 2]
```diff
@@ -24,19 +24,19 @@ import Foundation
 public class UserKYC: Codable {
     
     /// Object to hold information of  date of birth
-    public var dateOfBirth: DateOfBirth?
+    internal var dateOfBirth: DateOfBirth?
```
Comment (dipeshfrollo) [inline Sources/FrolloSDK/Model/Transient/UserKYC.swift:27]: why internal? How do we set these from client app?

File: Tests/Resources/API/User/user_details_complete.json [hunk 3]
```diff
@@ -1,7 +1,8 @@
 {
 	"id": 12345,
-	"first_name": "Jacob",
-	"last_name": "Frollo",
+	"given_name": "Jacob",
+	"family_name": "Frollo",
+    "middle_name": "",
```
Comment (dipeshfrollo) [inline Tests/Resources/API/User/user_details_complete.json:5]: can we test with some middle name ?

Comment (hailefrollo) [reply Tests/Resources/API/User/user_details_complete.json:5]: sure, will update unit test.

File: Sources/FrolloSDK/Model/Transient/UserKYC.swift (hunk 2 above)
Comment (dipeshfrollo) [reply Sources/FrolloSDK/Model/Transient/UserKYC.swift:27]: No point keeping in SDK if you cant access it. Even if we keep it (assuming we might need in future), lets make them public.

Comment (hailefrollo) [reply Sources/FrolloSDK/Model/Transient/UserKYC.swift:27]: Chat with Deepak, he left those public and left a comment to tell devs not using it. I will do the same 👍

File: Sources/FrolloSDK/KYC/KYC.swift (hunk 1 above)
Comment (hailefrollo) [reply Sources/FrolloSDK/KYC/KYC.swift:66]: I reverted this and will make change in WA

**Review by dipeshfrollo — APPROVED** [review]

**Pull request merged**